packages/plugin-e2e/src/models/pages/VariablePage.ts

```diff
@@ -5,6 +5,7 @@ import { VariableEditPage } from './VariableEditPage';
 export class VariablePage extends GrafanaPage {
   constructor(readonly ctx: PluginTestCtx, readonly dashboard?: DashboardPageArgs) {
     super(ctx);
+    this.dashboard = dashboard;
```

Comment:
Do you need this in a test? Think you can skip this line and add the `public` modifier before readonly on line 6.

Reply:
Have corrected this!